app/providers/database_service_provider.go

```diff
@@ -0,0 +1,22 @@
+package providers
+
+import (
+	"goravel/database/seeders"
+
+	"github.com/goravel/framework/contracts/database/seeder"
+	"github.com/goravel/framework/contracts/foundation"
+	"github.com/goravel/framework/facades"
+)
+
+type DatabaseServiceProvider struct {
+}
+
+func (receiver *DatabaseServiceProvider) Register(app foundation.Application) {
+	facades.Seeder().Register([]seeder.Seeder{
+		&seeders.DatabaseSeeder{},
+	})
```

Comment:
S: Move these lines to the `Boot` method, because all facades will be injected when calling `Boot`.